src/dlgAboutDialog.cpp

```diff
@@ -909,6 +909,10 @@ void dlgAboutDialog::setThirdPartyTab(const QString& htmlHead) const
     QString QtKeyChainHeader(tr("<h2><u>QtKeyChain - Platform-independent Qt API for storing passwords securely</u></h2>"
                                  "<h3>Copyright © 2011-2019 Frank Osterfeld &lt;frank.osterfeld@gmail.com&gt;.</h3>"));
 
+    QString SingleConnectHeader(tr("<h2><u>singleshot_connect.h - part of KDToolBox</u><br>(https://github.com/KDAB/KDToolBox).</h2>"
```

Comment:
Ah, let's make that URL clickable:
```suggestion
    QString SingleConnectHeader(tr("<h2><u>singleshot_connect.h - part of KDToolBox</u><br>(<a href=\"https://github.com/KDAB/KDToolBox\">https://github.com/KDAB/KDToolBox</a>).</h2>"
```

Reply:
I think this is enough for the about dialog. The aesthetic does not fit.

Reply:
And it doesn't work as I expect when I tried it - doing that and clicking on the link tried to open it in the dialogue instead of sending it to the system's default web-browser. Must be because the widget itself is one that can handle HTML documents...

So - yeah, forget that suggestion.